io.emeric.toolblex.yml

```diff
@@ -0,0 +1,260 @@
+app-id: io.emeric.toolblex
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: toolblex
+
+rename-desktop-file: toolblex.desktop
+rename-appdata-file: toolblex.appdata.xml
+rename-icon: toolblex
+
+finish-args:
+  # Our UI is GPU accelerated
+  - --device=dri
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # We need Bluetooth support
+  - --allow=bluetooth
+  - --system-talk-name=org.bluez
+  - --share=network
+
+cleanup:
+  - /bin/__pypache__
+  - /bin/rst*
+  - /include
+  - /lib/cmake
+  - /lib/cups
+  - /lib/pkgconfig
+  - /lib/python*
+  - /share/doc
+  - /share/man
+  - /share/zsh
+  - /src
+  - '*.a'
+  - '*.la'
+
+build-options:
+  append-ld-library-path: /app/lib
+
```

Comment:
```suggestion
```

This is unecessary, it's in the default path.

Reply:
Sure, I'll remove it for the next revision.